jenkins/pipelines/node-linter.jenkinsfile

```diff
@@ -42,15 +42,23 @@ pipeline {
             steps {
                 // Calling with `returnStatus` suppresses automatic failures
                 sh(script: "${env.MAKE} lint-md-build", returnStatus: true)
+                sh(script: "${env.MAKE} lint-py-build", returnStatus: true)
             }
         }
          
         stage('Run tests') {
             steps {
                 script {
+                    def node_version = sh(script: "python  tools/getnodeversion.py", returnStdout: true).trim()
+                    echo node_version
+                    def lint_py3_ret = 0
+                    if (node_version > "11." && node_version.take(1) != '8') {
```

Comment:
I think doing a `>` comparison on a String is not a good idea. Suggest something like the following that splits the version string into a list and then converts the first element in the list to an int:
```groovy
def version_parts = node_version.tokenize('.')
def major = version_parts[0] as int
if (major > 11) {
...
```

Reply:
Ack